flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/casting/CastRulesTest.java

```diff
@@ -1179,6 +1209,28 @@
         public Long b;
         public String c;
         public String[] d;
+
+        public MyStructuredType(long a, Long b, String c, String[] d) {
+            this.a = a;
+            this.b = b;
+            this.c = c;
+            this.d = d;
+        }
+
+        @Override
+        public String toString() {
```

Comment:
remove the `toString` again

Reply:
I wanna keep it, as it validates that this is not invoked (at least with the existing code)